lib/ru_propisju.rb

```diff
@@ -325,6 +380,18 @@ module RuPropisju
       :eur_fraction => ["євроцент", "євроцента", "євроцентів"],
       :usd_integral => ["долар", "долара", "доларів"],
       :usd_fraction => ['цент', 'цента', 'центів'],
+      :chf_integral => ['швейцарський франк', 'швейцарських франків', 'швейцарських франків'],
+      :chf_fraction => ['сантим', 'сантима', 'сантимів'],
+      :cny_integral => ['юань', 'юаня', 'юанів'],
+      :cny_fraction => ['фень', 'феня', 'феней'],
+      :gbp_integral => ['фунт стерлінгів', 'фунта стерлінгів', 'фунтів стерлінгів'],
+      :gbp_fraction => ['пенс', 'пенса', 'пенсів'],
+      :hkd_integral => ['гонконгський долар', 'гонконгського долара', 'гонконгських доларів'],
+      :hkd_fraction => ["цент", "цента", "центів"],
+      :jpy_integral => ['ієна', 'ієни', 'єн'],
```

Comment:
Not quite right, I'll leave the correct option below
```suggestion
      :jpy_integral => ['єна', 'єни', 'єн'],
```